src/commands/nfts/getJWT.ts

```diff
@@ -22,6 +22,7 @@ export const getJWT = async (
   
   try {
     ddo = await nvm.assets.resolve(did)
```

Comment:
is there a need for this here? it seems we just need the did

Reply:
true, removing it